snapcraft.yaml

```diff
@@ -160,20 +157,24 @@ parts:
       - libmbim-proxy
       - libmbim-utils
       - libgudev-1.0-0
+      - libqrtr-glib0
 
     override-build: |
       set -ex
+      PREV="$(pwd)"
```

Comment:
Which is the default folder now? BUILD folder? Maybe we can use CRAFT_PART_BUILD then directly instead of storing the folder in this variable.

Reply:
Yea this is the case, switched to that per your suggestion!